### PR TITLE
JENKINS: Don't run ucp_hello_world if cannot create endpoint with ucx_info

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6822/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@alinask @hoopoepg pls take a look (makes tests pass bgate w/o IB)

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -405,6 +405,11 @@ run_hello() {
 # Compile and run UCP hello world example
 #
 run_ucp_hello() {
+	if ./src/tools/info/ucx_info -e -u twe|grep ERROR
```
Comment (alinask) [inline contrib/test_jenkins.sh]: is the last 'e' needed? I get no valid output when i run it...

Comment (yosefe) [reply contrib/test_jenkins.sh]: yes, part of this PR is to add support for 'e' which means error handling

Comment (alinask) [reply contrib/test_jenkins.sh]: oh you added it, i see now :)

File: src/tools/info/ucx_info.c [hunk 2]
```diff
@@ -29,6 +29,7 @@ static void usage() {
     printf("                'r' : remote memory access\n");
     printf("                't' : tag matching \n");
     printf("                'w' : wakeup\n");
+    printf("                'e' : error handling\n");
```
Comment (hoopoepg) [inline src/tools/info/ucx_info.c]: minor: shouldn't arguments be literal-sorted? (e-error after a-atomic)

Comment (yosefe) [reply src/tools/info/ucx_info.c]: i wanted to put context options before ep options..

**Comment by alinask** [comment]
:+1:

**Pull request merged**